disdat/path_cache.py

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright 2015, 2016, 2017  Human Longevity, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Disdat Luigi Interface file
+
+The path cache keeps a binding from luigi tasks to bundles.
+
+"""
+
+import collections
+
+from disdat import logger as _logger
+
+PathCacheEntry = collections.namedtuple('PathCacheEntry', 'instance bundle uuid path rerun')
+
+
+class PathCache(object):
+    """
+    PipeCache singelton
+
+    The path cache requires that everyone import "disdat.path_cache" and not "import path_cache".
```

Comment:
is it possible to pull this import up into the top-level disdat.__init__?
This applies to all kinds of things in disdat

in general, it is bad form to have people importing nested modules.  they should just import disdat and get disdat.api, disdat.path_cache, etc for free